server/database/quiries/getUserByEmail.js

```diff
@@ -0,0 +1,3 @@
+const connection = require('../connection');
+
+module.exports = email => connection.query('SELECT * FROM users WHERE users.email = 1$', [email]).then(res => res.rows);
```

Comment:
I don't know if this will work as intended when exported.  Chaining should happen on the next line here.

If it gives you some asynchronous issues remove the ```.then``` from it. Otherwise there's no problem.